### PR TITLE
Checks if Windows is used

### PR DESCRIPTION
**Pull request by hale6666**
Also fixed typos

### CODE REVIEW

**Comment by hale6666** [comment]
Just checked, os.name() returns 'nt' when using Wine, so it's fine.

File: tts/sapi.py [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 """
-This ia a helper to use the Microsoft SAPI
+This is a helper to use the Microsoft SAPI
 
-Needs to run on a windows system
+Needs to run on a Windows system
 Requires comtypes
 """
 
+from os import name # Allows checking if using Windows
```
Comment (DeepHorizons) [inline tts/sapi.py]: Don't pollute the namespace with a commonly used variable. Instead import `os` and use `os.name`

File: tts/sapi.py [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+try:
+    assert(name() == 'nt') # Checks for Windows
```
Comment (DeepHorizons) [inline tts/sapi.py]: `name` is a string, not a method

**Comment by hale6666** [comment]
Sorry, I'm dumb.  Those issues should be fixed now

File: tts/sapi.py (hunk 2 above)
Comment (hale6666) [reply tts/sapi.py]: fixed

File: tts/sapi.py (hunk 1 above)
Comment (hale6666) [reply tts/sapi.py]: fixed

**Pull request merged**